lib/rouge/lexers/rego.rb

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+# example file taken from https://github.com/open-policy-agent/contrib/blob/f9e71d7/api_authz/docker/policy/api_authz.rego
+
+module Rouge
+    module Lexers
+        class Rego < RegexLexer
+            title "Rego"
+            desc "The Rego open-policy-agent (OPA) policy language (https://www.openpolicyagent.org/)"
+            tag 'rego'
+            aliases 'rego'
+            filenames '*.rego'
+
+            state :basic do
+              rule %r/\s+/, Text
+              rule %r/#.*/, Comment::Single
+      
+              rule %r/[\[\](){}|.,;!]/, Punctuation
+      
+              rule %r/"[^"]*"/, Str::Double
+      
+              rule %r/-?\d+\.\d+([eE][+-]?\d+)?/, Num::Float
```

Comment:
Is a number followed by a period (and nothing else, e.g. `13.`) treated as a number in Rego or as a number and punctuation?

Reply:
it'd be a parse error, actually, if used as a bare word. I should see if I can lex that as an error condition...

Reply:
No worries. It's a non-goal of Rouge to check for syntax correctness so you don't need to worry too much about ensuring syntactical mistakes are lexed as errors—just wanted to check this wasn't an overlooked corner case :)